api/v1/controllers/rsvpcontroller.js

```diff
@@ -8,13 +7,12 @@ export default {
 
     if (!meetup) return res.status(404).send({ status: 404, error: 'Meetup does not exist.' });
```

Comment:
In place of res.send, can you use res.json? Find out the difference and implement

Reply:
I have implemented the aforementioned changes